packages/webview_flutter/example/integration_test/webview_flutter_test.dart

```diff
@@ -1251,39 +1255,12 @@ void main() {
     expect(currentUrl, 'https://flutter.dev/');
   });
 
-  testWidgets('target _blank opens in same window',
```

Comment:
The default behavior for Android WebView is to ignore calling load on any non `https://` url. I tested this manually by calling `loadUrl('about:blank')`, `loadUrl('www.google.com')`, and `loadUrl('www.flutter.dev')`